### PR TITLE
Common page template

### PR DESCRIPTION
**Pull request by dbrgn**
Based on #32.

The current state was horrible, we had about half a dozen copies of the same template. Adjusting the discuss link required adjusting the navigation in 6 files in an identical way.

Now there's a single `page` template. It uses the title for the actual h1 title, not for the h2 headline below the navigation. If you need a different headline there, just use a `##` block.

The structure is still not ideal, but with less repetition.

I hope I didn't break anything, but it seems to work as before.

### CODE REVIEW

**Comment by ubruhin** [comment]
@dbrgn could you rebase this?

**Comment by dbrgn** [comment]
Done!

File: layouts/page/list.html [hunk 1]
```diff
@@ -1,12 +1,13 @@
+{# This is a hack to support subpages! Template is identical to single.html #}
```
Comment (ubruhin) [inline layouts/page/list.html]: Unfortunately this hack is visible in the rendered output :grin:

**Comment by dbrgn** [comment]
I give up, can't find out how this is supposed to work. Since now a `layouts/page/list.html` file exists, the home page uses that to display the content, and not `layouts/_default/list.html` anymore.

According to https://gohugo.io/templates/lookup-order/, if a `layouts/index.html` file exists, it should use that for the home page, however, that doesn't seem to work.

Usually there should also be a content page for the home page (something like `content/_index.md`), however that doesn't exist either in the LibrePCB website setup.

**Pull request closed without merging**